TestFoundation/TestFileManager.swift

```diff
@@ -1769,6 +1800,7 @@ VIDEOS=StopgapVideos
             ("test_getRelationship", test_getRelationship),
             ("test_displayNames", test_displayNames),
             ("test_getItemReplacementDirectory", test_getItemReplacementDirectory),
+            ("test_contentsEqual", test_contentsEqual),
```

Comment:
I was _very_ excited when the contentsEqual test didn't fail when I ran the suite... until I figured out why.